docs/send-data/kubernetes/install-helm-chart.md

```diff
@@ -248,6 +248,12 @@ otellogs:
       changeowner:
         securityContext:
           privileged: true
+opentelemetry-operator:
+  admissionWebhooks:
+    create: false
```

Comment:
@samiura I don't think we need this anymore after addressing the k8s label issue.